Week 4/Lecture 08/Assignment 03/assignment3/src/main/java/aliramadhan/assignment3/config/DataSourceConfig.java

```diff
@@ -0,0 +1,37 @@
+package aliramadhan.assignment3.config;
+
+
+import javax.sql.DataSource;
+
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.boot.jdbc.DataSourceBuilder;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class DataSourceConfig {
+
+
+    @Bean
+    @Qualifier("dataSource1")
+    public DataSource dataSource1() {
+        DataSourceBuilder dataSourceBuilder = DataSourceBuilder.create();
+        dataSourceBuilder.driverClassName("com.mysql.cj.jdbc.Driver");
+        dataSourceBuilder.url("jdbc:mysql://localhost:3306/employeeApp?useSSL=false&serverTimezone=Asia/Jakarta");
```

Comment:
Should put this database info in prop file and get by @Value, not hardcode in this.